src/openrct2/ride/coaster/SingleRailRollerCoaster.cpp

```diff
@@ -13829,14 +13829,14 @@ namespace SingleRailRC
         }
     }
 
-    static void TrackRightLargeHalfLoopDown(
+    static void TrackLeftLargeHalfLoopDown(
         PaintSession& session, const Ride& ride, uint8_t trackSequence, uint8_t direction, int32_t height,
         const TrackElement& trackElement)
     {
         TrackRightLargeHalfLoopUp(session, ride, 6 - trackSequence, direction, height, trackElement);
```

Comment:
Would this need adjusting in the name too or is that shared code for either left and right pieces?

Reply:
There are 4 of these `Track(Left|Right)LargeHalf(Up|Down)` functions, one for each combination.

Since this is for some painting stuff, and the upwards left half-loop looks exactly the same as the downward right one, just viewed from the opposite angle (and vice-versa), the code for the "down" pieces just calls the code for the (visually identical) upper piece.

These functions are only called in the large switch statement in `GetTrackPaintFunction` and nowhere outside this file.

However, their names are based on the name of the to-be-painted `TrackElemType`. (See the [original file](https://github.com/OpenRCT2/OpenRCT2/blob/42ff4716172d07c4555c7519e9d91c379b1a95ed/src/openrct2/ride/coaster/SingleRailRollerCoaster.cpp#L17816-L17823).)

My first commit changed only the names of the `TrackElemType` constants (at their definition site and in all cases where the names are used), this commit now renames the SingleRailCoaster's specific painting functions whose names where based on the to-be-painted `TrackElemType` constants.

I'm doing the same for each coaster type in `src/openrct2/ride/coaster` that has a specific painting function for the half-loop element (there are like 5 or so), each in its own commit.

---

Will later squash some of the commits, but for the moment I try to split this refactor into as small bits as possible (while keeping the code fully functional at each commit).